src/app/state/rest/index.ts

```diff
@@ -98,6 +99,16 @@ export const useBAGWithZipCode = (bagId: string, options?: Options) => {
   })
 }
 
+export const useBAGLodging = (type: string | undefined, subTypeId: string | undefined, options?: Options) => {
+  const handleError = useErrorHandler()
+  return useApiRequest<BAGObjectResponse>({
+    url: `https://api.data.amsterdam.nl/bag/v1.1/${ type }/${ subTypeId }/`,
```

Comment:
Although it's not being called. I would be more explicit here: `slashSandwich(["https://api.data.amsterdam.nl/bag/v1.1", type, subTypeId], { trailingSlash: true })`. slash-sandwich filtered `undefined`.

Reply:
Done